patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
++            return enumdirection.g;
++        }));
++        p = (EnumDirection[]) list.toArray(new EnumDirection[0]);
++    }
++
++    private static final EnumDirection[] q;
++
++    static {
++        List<EnumDirection> list = new ArrayList<>();
++        for (EnumDirection enumDirection : EnumDirection.n) {
++            if (enumDirection.n().d()) {
++                list.add(enumDirection);
++            }
++        }
++        list.sort(Comparator.comparingInt((enumdirection) -> {
++            return enumdirection.i;
++        }));
++        q = (EnumDirection[]) list.toArray(new EnumDirection[0]);
++    }
++
++    private static final Long2ObjectMap<EnumDirection> r;
++
++    static {
++        Long2ObjectOpenHashMap<EnumDirection> map = new Long2ObjectOpenHashMap<>();
++        for (EnumDirection enumDirection : EnumDirection.n) {
++            map.merge((new BlockPosition(enumDirection.p())).asLong(), enumDirection, (enumdirection, enumdirection1) -> {
++                throw new IllegalArgumentException("Duplicate keys");
++            });
++        }
++        r = (Long2ObjectMap) map;
++    }
+ 
+     private EnumDirection(int i, int j, int k, String s, EnumDirection.EnumAxisDirection enumdirection_enumaxisdirection, EnumDirection.EnumAxis enumdirection_enumaxis, BaseBlockPosition baseblockposition) {
+         this.g = i;
+@@ -340,9 +365,18 @@ public enum EnumDirection implements INamable {
+ 
+         private static final EnumDirection.EnumAxis[] e = values();
+         public static final Codec<EnumDirection.EnumAxis> d = INamable.a(EnumDirection.EnumAxis::values, EnumDirection.EnumAxis::a);
+-        private static final Map<String, EnumDirection.EnumAxis> f = (Map) Arrays.stream(EnumDirection.EnumAxis.e).collect(Collectors.toMap(EnumDirection.EnumAxis::b, (enumdirection_enumaxis) -> {
+-            return enumdirection_enumaxis;
+-        }));
++        private static final Map<String, EnumDirection.EnumAxis> f;
++
++        static {
++            Map<String, EnumAxis> map = new HashMap<>();
++            for (EnumAxis enumdirection_enumaxis : EnumAxis.e) {
++                if (map.put(enumdirection_enumaxis.b(), enumdirection_enumaxis) != null) {
++                    throw new IllegalStateException("Duplicate key");
++                }
++            }
++            f = (Map) map;
++        }
++
+         private final String g;
+ 
+         private EnumAxis(String s) {
+diff --git a/src/main/java/net/minecraft/server/EnumProtocol.java b/src/main/java/net/minecraft/server/EnumProtocol.java
+index ec3c8ec6a7c0a835d0bab9872d0ead68b2ff54df..b3cee9bc82aacba99f91ffb4c955ed5321e8250f 100644
+--- a/src/main/java/net/minecraft/server/EnumProtocol.java
++++ b/src/main/java/net/minecraft/server/EnumProtocol.java
+@@ -65,15 +65,17 @@ public enum EnumProtocol {
+             }
+ 
+             EnumProtocol.e[k - -1] = enumprotocol;
+-            enumprotocol.h.forEach((enumprotocoldirection, enumprotocol_a) -> {
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/util/Map.java#l618